helm_deploy/values-prod.yaml

```diff
@@ -20,3 +20,4 @@ env:
   INTERVENTIONSUI_BASEURL: "https://refer-monitor-intervention.service.justice.gov.uk"
   COMMUNITYAPI_BASEURL: "https://community-api-secure.probation.service.justice.gov.uk"
   SENTRY_ENVIRONMENT: "prod"
+  NOTIFY_ENABLED: false
```

Comment:
how about preprod?

Reply:
I assumed we wouldn't want to send emails to real providers on pre-prod, so that wouldn't be an issue. Is that correct?